docs/quickstart.md

```diff
@@ -37,14 +22,39 @@ npm install --save firebase reactfire
 
    ![new document screenshot](https://firebasestorage.googleapis.com/v0/b/rxfire-525a3.appspot.com/o/docs%2FScreen%20Shot%202019-07-03%20at%202.19.11%20PM.png?alt=media&token=052d27ea-5db1-4a02-aad0-a3f017c1a975)
 
-1. Add the following to your security rules and click _Publish_
+1. Add security rules to your document.
 
+    1. In the _Rules_ tab of the console, add the following security rules:
+   
    ```text
    match /tryreactfire/burrito {
      allow read: if true;
      allow write: if request.auth.uid != null;
    }
```

Comment:
@shelcindra just a heads up - I saw the comment about new security rules, but looks like they haven't changed yet

Reply:
Oops!  Checked it off too soon!  Added now.